samples/net/coaps_server/prj_qemu_x86.conf

```diff
@@ -14,7 +14,7 @@ CONFIG_NET_PKT_TX_COUNT=14
 CONFIG_NET_BUF_RX_COUNT=30
 CONFIG_NET_BUF_TX_COUNT=30
 CONFIG_NET_IF_UNICAST_IPV6_ADDR_COUNT=3
-CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=2
+CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=5
```

Comment:
Can we provide some safe and generous defaults for this type of low level configuration options so we do not have to deal with them in sample configurations and just take the default?